api/server/server.go

```diff
@@ -152,13 +153,16 @@ func NewServer(conf Config) (*Server, error) {
 	}
 
 	var err error
-	clientBuilder, err := lotus.NewBuilder(conf.LotusAddress, conf.LotusAuthToken, lotusConnectionRetries)
+	clientBuilder, err := lotus.NewBuilder(conf.LotusAddress, conf.LotusAuthToken, conf.LotusConnectionRetries)
 	if err != nil {
 		return nil, fmt.Errorf("creating lotus client builder: %s", err)
 	}
-	lotus.MonitorLotusSync(clientBuilder)
+	lsm, err := lotus.NewSyncMonitor(clientBuilder)
```

Comment:
What before was some background running daemon that generated Prometheus metrics for the grafana dashboard regarding Lotus sync height, now is a more formal component to provide Lotus syncing information for decision making.